deltachat-ios/en.lproj/Localizable.strings

```diff
@@ -896,3 +896,5 @@
 "webxdc_empty_hint" = "Received or sent apps will appear here. Tap \"Files\" to select downloaded apps.";
 "shortcut_share_btn" = "Click the share button";
 "shortcut_add_to_home_description" = "Select - Add to Home Screen - to add the app to your home screen.";
+"connectivity_low_data_mode" = "Low Data Mode: background message fetching is disabled.";
```

Comment:
maybe 

```suggestion
"connectivity_low_data_mode" = "Disabled by system's \"Low Data Mode\"";
```

this fits better as it basically reads "Notifications: Disabled ..." - and also the other strings are similar ("Disabled in Delta Chat" or "Disabled om system settings").

and yes, according to https://support.apple.com/en-us/HT210596 it is actually disabled.